cmake/features.cmake

```diff
@@ -5,7 +5,7 @@
 #
 # Common cmake options
 #
-ov_option (ENABLE_PROXY "Proxy plugin for OpenVINO Runtime" ON)
+ov_option (ENABLE_PROXY "Proxy plugin for OpenVINO Runtime" OFF)
```

Comment:
maybe we need to do it only for Release branch?
In this case we still can test this functionality

Reply:
Yes, upon release branch would be created pr will be put here

Reply:
Yes, I think after the release branch creation we need to switch the base branch for the RR